portfolio/src/main/java/com/google/sps/servlets/Comment.java

```diff
@@ -0,0 +1,80 @@
+package com.google.sps.servlets;
+
+/**
+ * Encapsulated the possible values for reasons a user visited a site.
+ * Update this enum when adding or removing allowable Visit Type values to the
+ * Comments form.
+ *
+ * NONE represents a null response (i.e. user did not answer the question)
+ * All other responses correspond with one radio button on the form
+ */
+enum VisitType {
+  NONE, RECRUITING, PROJECT, TUTORING, CHAT
+}
+
+/**
+ * Encapsulates a user-entered comment and their personal information,
+ * including their first and last name, reason for visiting, and email address.
+ *
+ * Can only be initialized once.
```

Comment:
That's a situation in which `final` comes in handy. It allows you to use the language to statically enforce the constraint you are mentioning